pkg/lib/clusterconfig/clusterconfig.go

```diff
@@ -391,6 +396,18 @@ func (cc *Config) Validate(accessKeyID string, secretAccessKey string) error {
 		return errors.Wrap(err, InstanceTypeKey)
 	}
 
+	zones, err := aws.GetAvailabilityZones(accessKeyID, secretAccessKey, *cc.Region)
```

Comment:
To save a little bit of time / potential for unexpected errors, we can run this check only if `len(cc.AvailabilityZones) > 0`